.github/workflows/NotEnoughUpdates-REPO-Workflow.yml

```diff
@@ -2,8 +2,6 @@ name: NotEnoughUpdates REPO Workflow
 
 # Controls when the action will run. 
 on:
-  pull_request_target:
```

Comment:
You know you can just detect if it is a pull request inside one workflow instead of needing two right

Reply:
Yeah but I don’t know how to change “using” depending on that, this is why you’re still workflow man

Reply:
The easy way is to make two run-checks that both have an if option 
```yml
if: ${{ github.event_name == "pull_request_target" }}
using: whatever
```
then add the other one with == "push"